github_connector/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,9 @@
+* Sylvain LE GAL (https://twitter.com/legalsylvain)
+* Sébastien BEAU (sebastien.beau@akretion.com)
+* Benoît GUILLOT (benoit.guillot@akretion.com)
+* Enrique Martín (enriquemartin@digital5.es)
+
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+    * Vicent Cubells
+    * Carlos Roca
```

Comment:
Please include me here, because I have contributed (you can check the history :wink: ), but forgot it to add me.